src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -61,6 +64,88 @@ export default class MutationMapper extends React.Component<IMutationMapperProps
         };
     }
 
+    @computed get mutationRateSummary() {
+        if (this.props.store.gene.result &&
+            this.props.store.patientIds.result &&
+            this.props.store.mutationData.result &&
+            this.props.store.mskImpactGermlineConsentedPatientIds.result) {
+
+            const germlineMutationRate = this.germlineMutationRate(this.props.store.mutationData.result,
+                                                                   this.props.store.patientIds.result,
+                                                                   this.props.store.mskImpactGermlineConsentedPatientIds.result)
+            const germlineMutationRateElement:JSX.Element|null =
+                (this.props.store.mskImpactGermlineConsentedPatientIds.result.length
+                    > 0 || germlineMutationRate > 0)?
+                (
+                    <span style={{paddingRight: 5}}>
+                     Germline Mutation Rate:
+                         <span style={{paddingLeft:2}}>
+                            {germlineMutationRate.toFixed(1)}%,
+                         </span>
+                    </span>
+                )
+                : null;
+
+            return (
+                <h4 style={{paddingBottom:2,paddingTop:2}}>
+                    {this.props.store.gene.result.hugoGeneSymbol}:
+                         [{germlineMutationRateElement}
+                      Somatic Mutation Rate:
+                         <span style={{paddingLeft:2}}>
+                         {this.somaticMutationRate(this.props.store.mutationData.result,
+                                                   this.props.store.patientIds.result).toFixed(1)}%]
+                         </span>
+                </h4>
+            );
+        } else {
+            return '';
+        }
+    }
+
+   private germlineMutationRate(mutations: Mutation[],
+       patientIds: string[],
+       mskImpactGermlineConsentedPatientIds: string[]) {
+
+       const nrCasesGermlineMutation:number =
+           _.chain(this.props.store.mutationData.result)
```

Comment:
make this "functional" by passing in the mutationData instead of referencing it on "this".  That way this function would either be static method or even just a function outside of the class.